tests/test_main.py

```diff
@@ -6,18 +6,19 @@
 from app.point import Point
 
 
-# @pytest.mark.timeout(5)
-# def test_deletion():
-#     items = [(f"Element {i}", i) for i in range(1000)]
-#     dictionary = Dictionary()
-#     for key, value in items:
-#         dictionary[key] = value
-#     for key, value in items:
-#         assert dictionary[key] == value
-#     assert len(dictionary) == len(items)
-#     for key, value in items:
-#         del dictionary[key]
-#     print(len(dictionary))
+@pytest.mark.timeout(5)
```

Comment:
no need to push changes to the tests to remote

Reply:
I implemented the method for deleting a node, so I assumed it needed to be uncommented if I completed it.